src/screens/dashboard/components/Carousel.tsx

```diff
@@ -3,14 +3,20 @@ import { User } from 'mock-api/models/mirageTypes'
 import React, { useMemo, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { TouchableOpacity } from 'react-native'
-import { FlatList } from 'react-native-gesture-handler'
 import { CarouselElement } from 'screens/dashboard/components/CarouselElement'
 import { getCurrentLocale } from 'utils/locale'
 import { Text } from 'utils/theme'
 import { useSortAllHolidayRequests } from 'utils/useSortAllHolidayRequests'
 import { Analytics } from 'services/analytics'
+import { FlashList } from '@shopify/flash-list'
 import { TeamMemberModal } from '../DashboardTeam'
 
+const CAROUSEL_ITEM_WIDTH = 94.2
+
+interface FlatListItem {
```

Comment:
is there any special purpose for interface?

Reply:
Nope, im just used to them from TB, I will change to type :D